newIDE/app/src/EventsSheet/index.js

```diff
@@ -939,10 +1039,19 @@ export class EventsSheetComponentWithoutHandle extends React.Component<
     ) {
       return;
     }
-
-    this._saveChangesToHistory(() => {
-      if (this._eventsTree) this._eventsTree.forceEventsUpdate();
-    });
+    const positions = this._getChangedEventRows(
+      getSelectedEvents(this.state.selection)
+    );
+    this._saveChangesToHistory(
+      'ADD',
+      {
+        previous: positions,
```

Comment:
should we rename these props to be more explicit? it took me a bit of time to understand how they differ if they are the same.
Something like `positionsBeforeAction` and `positionsAfterAction` ?

Reply:
also, do we care about all positions? should this just be `position` and we do the logic here to select which one we consider as the most important? (the top one I'd say?)